### ISSUE TITLE
DeviceLab Windows tests execution time doubled in the past two weeks

### ISSUE THREAD

**Issue by keyonghan** [flutter/flutter#68058]
From bi-weekly flutter infra metrics/slo review meeting, we find execution time of DeviceLab Windows tests doubled, compared to two weeks earlier:
90th% execution time: 09/14-09/27 **4.77** min -> 09/28-10/11 **8.89** min

**Comment by jonahwilliams** [comment]
Does this include downloading dependencies? For example, most of the recent flutter_gallery_win__compile tasks seem to take either 2 or 5 minutes, but the sheet lists almost 10 minutes

**Comment by keyonghan** [comment]
Yes and no. If test runs with same existing flutter sha on the agent, then it will reuse previously checked out flutter revision, and run directly without downloading, otherwise it will do checkout and dependencies downloading.

Note the numbers in the sheet are 90th% values.
